JS_1_pow.js

```diff
@@ -0,0 +1,26 @@
+var x = prompt('Введите число x:');
+var n = prompt('Укажите степень n (целое число):');
+

```

Comment:
Please add entered data validation if required by the task.

Reply:
is not required now. Will be done in later version